src/utils/ticketFinder/__tests__/ticketFinder.test.js

```diff
@@ -46,12 +46,29 @@ describe('ticketFinder', () => {
   });
 
   it('should find Jira ticket when there are tickets in the branch name only', async () => {
-    const tickets = await ticketFinder({ body: mockBodyWithNothing, head: { ref: 'therynamo/JIRA-123' } });
+    // ticket id only
+    var tickets = await ticketFinder({ body: mockBodyWithNothing, head: { ref: 'JIRA-123' } });
```

Comment:
```suggestion
    let tickets = await ticketFinder({ body: mockBodyWithNothing, head: { ref: 'JIRA-123' } });
```

Reply:
The linter should have caught this, but usually we want to "prefer [`let`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let)".

Reply:
Me too. Fixed it.